EnglishTranslation-master/SKILL.tsv

```diff
@@ -5999,7 +5999,7 @@ SKILL_20151224_005998	5% increased EXP rate from automatic Instanced Dungeon mat
 SKILL_20151224_005999	Token Benefits: Movement Speed Increase
 SKILL_20151224_006000	Increased movement speed
 SKILL_20151224_006001	Token
-SKILL_20151224_006002	 - Able to learn up to 2 attributes simultaneously{nl} - Reduced Market commission fee (30%{img white_right_arrow 16 16}10%){nl} - Able to list up to 5 items on the Market{nl} - +3 increased character movement
+SKILL_20151224_006002	Able to learn up to 2 attributes simultaneously{nl}Reduced Market commission fee (30%{img white_right_arrow 16 16}10%){nl}Able to list up to 5 items on the Market{nl}+3 increased character movement
```

Comment:
The '-' signs are there for a reason, so that it is easier to read. Revert this pelase.